nexus-sdk-js/tsconfig.json

```diff
@@ -0,0 +1,68 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "esnext",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```

Comment:
I am not sure about browser support for ESNEXT, let's discuss

Reply:
Absolutely, can be anything. It doesn't directly impact the code.